lib/list.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'order.rb'
+
+class List
+  
+  attr_reader :menu, :order
+  
+  def initialize
+    @menu = {
+      'apple' => 3,
+      'ham' => 4
+     }
+     @order =[]
+  end
+  
+  def method_missing(m, *args, &blk)
```

Comment:
A little confused about these argument names and what they are actually representing.